addons/account/static/src/js/reconciliation/reconciliation_model.js

```diff
@@ -1083,6 +1084,7 @@ var StatementModel = BasicModel.extend({
             'tax_id': this._formatNameGet(values.tax_id),
             'debit': 0,
             'credit': 0,
+            'date': values.date ? values.date : moment(),
```

Comment:
Be careful when working with date, when you create a date using moment() like this, it will be created with the timezone of the user, and later on you send the date directly to the server without changing it, which might be problematic as we expect a date in a given format and in UTC. There are some methods that can be use in the js framework to work with date. For example: in 'field_utils.js' there is a method called: 'parseDate' which takes a string as input and will return a moment Object in the locale of the user but we overwrite the toJSON method of that Object in order to send the correct date back to the server. So what you can do is for example use the following:
field_utils = require('web.field_utils');
var today = new moment().utc().format();
'date' : values.date ? values.date : field_utils.date.format(today, {}, {isUTC: true})

That way you don't have to worry about timezone as it will be displayed in the view in the timezone of the user and will works fine when we send back the value to the server as it will be converted using the custom toJSON method automatically.

Note that it is possible that it was working due to luck because when we instanciate the Date field in the renderer, it is possible that change to that field uses a correct modified moment Object instead of the one you've created here (haven't tested it), but it would be better to change this in case we later have to overwrite or change this method and for some unknown reason we would want to bypass the renderer and use the value directly.